pkg/kubehound/graph/edge/exploit_host_traverse_token.go

```diff
@@ -17,13 +17,13 @@ import (
 
 // TokenMountList represent ounts that grant access to the pod service account tokens that reside
 // in /var/lib/kubelet/pods/<uid>/volumes/kubernetes.io~projected/<name>/
-var TokenMountList = []primitive.Regex{
-	{Pattern: "^/$"},
-	{Pattern: "^/var$"},
-	{Pattern: "^/var/lib$"},
-	{Pattern: "^/var/lib/kubelet$"},
-	{Pattern: "^/var/lib/kubelet/pods$"},
-	{Pattern: "^/var/lib/kubelet/pods/.*"},
+var TokenMountList = bson.A{
+	"/",
+	"/var",
+	"/var/lib",
+	"/var/lib/kubelet",
+	"/var/lib/kubelet/pods",
+	// Omit further depth matching as unlikely to occur and requires a slow regex match
```

Comment:
What is the performance impact of `^/var/lib/kubelet/pods/.*` here? couple seconds ? or more?

Reply:
its probably fine actually - will add it back in